functions/Enable-DbaTraceFlag.ps1

```diff
@@ -1,21 +1,21 @@
 function Enable-DbaTraceFlag {
 	<# 
 	.SYNOPSIS 
-		Enable a Global Trace Flag 
+		Enable Global Trace Flag(s)
 	.DESCRIPTION
 		The function will set one or multiple trace flags on the SQL Server instance(s) listed
 	
 	.PARAMETER SqlInstance
 		Allows you to specify a comma separated list of servers to query.
 
 	.PARAMETER SqlCredential
-		Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
-		$cred = Get-Credential, this pass this $cred to the param. 
-
-		Windows Authentication will be used if DestinationSqlCredential is not specified. To connect as a different Windows user, run PowerShell as that user.	
+		Allows you to login to servers using SQL Logins instead of Windows Authentication (AKA Integrated or Trusted). To use:
+		$scred = Get-Credential, this pass this $scred object to the -SqlCredential parameter.
+		Windows Authentication will be used if SqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.
+		To connect as a different Windows user, run PowerShell as that user.	
```

Comment:
We need the format of this section to include the new lines as this is the format we are standardizing to with our templates ([example](https://github.com/sqlcollaborative/dbatools-templates/blob/master/functions/Get-Noun.ps1)):

```			Allows you to login to servers using SQL Logins instead of Windows Authentication (AKA Integrated or Trusted). To use:



			$scred = Get-Credential, then pass $scred object to the -SqlCredential parameter.



			Windows Authentication will be used if SqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.



			To connect as a different Windows user, run PowerShell as that user.